application/common/config/main.php

```diff
@@ -193,6 +193,7 @@
         'idpName'                       => $idpName,
         'idpDisplayName'                => $idpDisplayName,
         'mfaNagInterval'                => Env::get('MFA_NAG_INTERVAL', '+30 days'),
+        'methodNagInterval'             => Env::get('METHOD_NAG_INTERVAL', '+30 days'),
```

Comment:
Can we change the default to `+6 months`?
```suggestion
        'methodNagInterval'             => Env::get('METHOD_NAG_INTERVAL', '+6 months'),
```